configuration/config.xml.tpl

```diff
@@ -32,6 +32,7 @@
     <permission>hudson.model.Item.Workspace:authenticated</permission>
     <permission>hudson.model.View.Read:authenticated</permission>
     <permission>jenkins.metrics.api.Metrics.View:authenticated</permission>
+    <permission>hudson.model.Hudson.Read:anonymous</permission>
```

Comment:
For my own understanding, is this needed at this level, even when we explicitly set that permission on the particular org we want to allow anonymous read on?

Reply:
I've just been playing around with this on an instance. I've manually changed over to project-based matrix authorisation strategy and it seems to work fine. @mikenairn if you don't have any objections, I'll rebase this and merge it, and manually change our existing instances.